wp-resources/plugins/sil-dictionary-webonary/webonary/Webonary_Cloud.php

```diff
@@ -370,7 +370,7 @@ public static function searchEntries($posts, WP_Query $query): ?array {
 				[
 					'key' => ['filter' => FILTER_UNSAFE_RAW],
 					'tax' => ['filter' => FILTER_UNSAFE_RAW],
-					'match_whole_words' => ['filter' => FILTER_UNSAFE_RAW, 'options' => ['default' => '1']],
+					'match_whole_words' => ['filter' => FILTER_UNSAFE_RAW, 'options' => ['default' => '0']],
```

Comment:
Does it matter that the default on the UI is match_whole_words=1? 
![image](https://user-images.githubusercontent.com/4266917/174388357-8f788adc-010a-49f0-a725-1737496a74b3.png)

Reply:
This says that if `match_whole_words` parameter is not specified in the URL, assume falsy, `'0'`. This matches the current behavior in the PHP side, https://github.com/sillsdev/webonary/blob/437cd8e3bf07fae53411997fed0ba6838f469e95/wp-resources/plugins/sil-dictionary-webonary/include/dictionary-search.php#L71